php-frontend/src/test/java/org/sonar/php/PHPAnalyzerTest.java

```diff
@@ -20,12 +20,14 @@
 package org.sonar.php;
 
 import com.sonar.sslr.api.RecognitionException;
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Set;
+import javax.annotation.Nullable;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
```

Comment:
I think this is the issue, why you don't see test coverage. You should use jupiter in this case. I'm also not sure if all old test files would now be covered if you ran a fresh sonar scan

Reply:
Indeed, it solved the missing coverage! Thanks!